src/index.js

```diff
@@ -12,4 +12,4 @@ addTodo();
 deletetodo();
 edittodo();
 interacttodo();
-clearcompleted();
\ No newline at end of file
+clearcompleted();
```

Comment:
- [ ] While some of the functionalities work, after marking an item as completed, the checkbox is not checked which does not indicate to the user the item has been completed. Please kindly fix this.

![image](https://github.com/kayjeee/webpack-todolist/assets/43995830/b1b776ad-dacf-4745-8c8f-8ecbdc439cbd)